code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -220,6 +220,22 @@
 	min_cold_protection_temperature = T0C - 20
 	siemens_coefficient = 0.7
 
+/obj/item/clothing/suit/storage/scavengerarmor
+	name = "Scavenger armor"
+	desc = "A sturdy, rigged Scavenger armor. strong and sturdy as most vests. made fully from junk."
+	icon_state = "leather_jacket"
+	item_state = "leather_jacket"
+	armor = list(
+		melee = 30,
+		bullet = 30,
+		energy = 30,
+		bomb = 10,
+		bio = 0,
+		rad = 0
+	)
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
```

Comment:
Not even the  basic ih armor vests get arm protection, that's probably too good for joonk vest.

Reply:
wow i didn't knew that. """wow IH vest sucks...""" i thought it was one of the best...

Reply:
overlooked it. well changes being made